### PR TITLE
Fixing Issue #829

### PR DESCRIPTION
**Pull request by Bernasss12**
### Trying to fix issue #829
`GuiRequestTable.java` has no hardcoded text.

Please review this commit and confirm/deny if this was the basis of the issue or whether this is to be done or not. 

Also please review the `en_us.lang` file formatting and correct me in any way you see fit.

Thanks for the attention.

### CODE REVIEW

File: resources/assets/logisticspipes/lang/en_us.lang [hunk 1]
```diff
@@ -323,6 +323,54 @@ lp.hud.config.satellite.enabled=Enabled Satellite.
...
+gui.request.text.message.missing=Missing
+gui.request.text.message.components=Components
+
+
+
+
+
+
+
+
```
Comment (bziemons) [inline resources/assets/logisticspipes/lang/en_us.lang]: empty lines?

**Comment by bziemons** [comment]
I don't know if we want to translate symbols.. It isn't bad, but it would make language files quite a bit bigger. Other than that you are on the right track.

File: resources/assets/logisticspipes/lang/en_us.lang (hunk 1 above)
Comment (Bernasss12) [reply resources/assets/logisticspipes/lang/en_us.lang]: Bad habits, I don't like to see text below where I'm writing but I'll make sure to always remove it before committing from now on.

**Comment by Bernasss12** [comment]
I was actually skeptical if those were indeed necessary to change since, even without thinking about language file size, I can't thing of a single reason to change it in other languages, and if changed it would probably only actually break the GUI. I'll fix that and not change these cases from now on.

About the lang file comments/organization/naming anything to point out?

**Comment by bziemons** [comment]
That's fine :)

> About the lang file comments/organization/naming anything to point out?

Looks good. Just keep in mind to re-use the translations if applicable and rename the keys to something more general (in that case) :)

**Comment by Bernasss12** [comment]
I had that in mind, in fact the class I finished can pretty much have everything re-used for the Request Pipes :)
But thanks for pointing out!

File: resources/assets/logisticspipes/lang/en_us.lang [hunk 2]
```diff
@@ -322,16 +322,48 @@ lp.hud.config.provider.disabled=Disabled Provider.
...
+gui.request.textUpperID=ID
+gui.request.textRequestType=Request Type
+gui.request.textSendRouterID=Send to Router ID
+gui.request.textRequestID=Request ID
+gui.request.textYouAreMissing=You are missing:
+gui.request.textRequest_success=Request successful!
+gui.request.textMissing=Missing
+gui.request.textComponents=Components
+# - ItemDisplay
+gui.item_displayp.age=Page
```
Comment (davboecki) [inline resources/assets/logisticspipes/lang/en_us.lang:354]: typo?

File: common/logisticspipes/utils/gui/ItemDisplay.java [hunk 3]
```diff
@@ -271,9 +284,12 @@ public void renderItemArea(double zLevel) {
 			buf.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION_TEX);
 			int xPosition = (width / 2) - 50;
 			int yPosition = 40;
-			buf.pos(xPosition, yPosition + 100, zLevel).tex(0.04, 0.72 + (graphic * 0.03125)).endVertex();;
-			buf.pos(xPosition + 100, yPosition + 100, zLevel).tex(0.08, 0.72 + (graphic * 0.03125)).endVertex();;
-			buf.pos(xPosition + 100, yPosition, zLevel).tex(0.08, 0.69 + (graphic * 0.03125)).endVertex();;
+			buf.pos(xPosition, yPosition + 100, zLevel).tex(0.04, 0.72 + (graphic * 0.03125)).endVertex();
+			;
```
Comment (davboecki) [inline common/logisticspipes/utils/gui/ItemDisplay.java:288]: Instead of creating new lines here, just remove them, please.

**Comment by Bernasss12** [comment]
I already changed that later on but yes that seems to be a typo

**Comment by davboecki** [comment]
Other than the two comments I added everything else lgtm.

File: common/logisticspipes/utils/gui/ItemDisplay.java (hunk 3 above)
Comment (Bernasss12) [reply common/logisticspipes/utils/gui/ItemDisplay.java:288]: I don't really know what happened here but I'm guessing it was the auto formatting

Comment (Bernasss12) [reply common/logisticspipes/utils/gui/ItemDisplay.java:288]: But yes Ill remove the semicolons

**Comment by Bernasss12** [comment]
I will restart my branch for this issue since there has been a lot of updates since I started this.

**Comment by bziemons** [comment]
just rebase and you'll be fine

**Comment by Bernasss12** [comment]
It's much easier and less prone to error this way for me

**Pull request closed without merging**